conf/messages.en

```diff
@@ -183,9 +198,10 @@ associate.ucr.summary.kind.ducr = DUCR
 associate.ucr.summary.addConsignment = Add consignment
 associate.ucr.summary.masterConsignment = To master consignment
 
+# TODO clean up next 3 lines after last page is done
```

Comment:
Is this still required? Should we remove it

Reply:
Yes, I need to clean up after fixing the other 3 pages.